source/clear-linux/get-started/virtual-machine-install/vmware-player-2.rst

```diff
@@ -0,0 +1,469 @@
+.. _vmware-player:
+
+Use VMware\* Workstation Player
+###############################
+
+This section explains how to run |CLOSIA| as a virtual machine using the 
+`VMware Workstation 14 Player`_ hypervisor.
+
+There are 2 ways to create a |CL| VM to run in VMware:
```

Comment:
Use the `:abbr:` label in this first instance of "VM":
:abbr:\`VM (Virtual Machine)`